cmd/minishift/cmd/start.go

```diff
@@ -154,42 +128,50 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	//Create the host directories if not present
-	hostDirs := []string{viper.GetString(startFlags.HostConfigDir.Name),
-		viper.GetString(startFlags.HostDataDir.Name), viper.GetString(startFlags.HostVolumeDir.Name),
-		viper.GetString(startFlags.HostPvDir.Name)}
-	err = clusterup.EnsureHostDirectoriesExist(libMachineClient, hostDirs)
-	if err != nil {
-		fmt.Println("Error creating required host directories: ", err)
-		atexit.Exit(1)
+	hostDirs := []string{
```

Comment:
Is clusterup.EnsureHostDirectoriesExist not the function you are after? The thing is, I need to provide this function with the set of directories to check/create. you want me to put 

	hostDirs := []string{
		viper.GetString(startFlags.HostConfigDir.Name),
		viper.GetString(startFlags.HostDataDir.Name),
		viper.GetString(startFlags.HostVolumeDir.Name),
		viper.GetString(startFlags.HostPvDir.Name),
	}

into a separate function?